WordPress/Classes/Networking/BlogServiceRemote.h

```diff
@@ -4,9 +4,14 @@
 
 typedef void (^OptionsHandler)(NSDictionary *options);
 typedef void (^PostFormatsHandler)(NSDictionary *postFormats);
+typedef void (^MultiAuthorCheckHandler)(BOOL isMultiAuthor);
 
 @protocol BlogServiceRemote <NSObject>
 
+- (void)checkMultiAuthorForBlog:(Blog *)blog
+                        success:(MultiAuthorCheckHandler)success
+                        failure:(void (^)(NSError *error))failure;
```

Comment:
I think this could be a good opportunity to add a typedef for the error block (`ErrorHandler` or something like that).  Would make this line so much friendlier.